docs/podman-tree.1.md

```diff
@@ -0,0 +1,37 @@
+% podman-image-tree(1)
+
+## NAME
+podman\-image\-tree - Show the image hierarchy in local image cache
+
+## SYNOPSIS
+**podman image tree** [*image*:*tag*]**|**[*image-id*]
+[**--help**|**-h**]
+
+## DESCRIPTION
+Displays the image hierarchy in local image cache. The information is printed in tree format
+If you do not provide *tag*, podman will default to `latest` for the *image*.
```

Comment:
No. It supports both ID and name. Basically it uses function [`ImageRuntime.NewFromLocal()`](https://github.com/containers/libpod/pull/1642/files#diff-7b896d973a999ecbc72614c92c79a560R59) , that means whatever ``podman image`` supports, this will also support.

My examples were with fully qualified names also. https://github.com/containers/libpod/pull/1642#issuecomment-457116705